### PR TITLE
Add react-native-vector-icons

### PR DESCRIPTION
**Pull request by Lapanti**
Related issue: https://github.com/futurice/pepperoni-app-kit/issues/40

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -1,3 +1,3 @@
 PODFILE CHECKSUM: cb3339de6360cad344c46bd08cc9861e861af500
 
-COCOAPODS: 1.1.1
+COCOAPODS: 1.0.1
```
Comment (tino-junge) [inline ios/Podfile.lock]: Let's not downgrade cocoapods here ;)

**Comment by tino-junge** [comment]
Thanks for the PR @Lapanti :) 🚀 Will check it tomorrow

File: android/app/src/main/java/com/pepperoniapptemplate/MainActivity.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 package com.pepperoniapptemplate;
 
 import com.facebook.react.ReactActivity;
+import com.oblador.vectoricons.VectorIconsPackage;
```
Comment (fabriziomoscon) [inline android/app/src/main/java/com/pepperoniapptemplate/MainActivity.java]: Actually this import needs to go in the `MainApplication.java` and this 
```
new VectorIconsPackage()
```
needs to be added inside the list of packages

**Comment by tino-junge** [comment]
@Lapanti, I found the problem for your failing tests. You need to set the version for `jest-react-native` to 17.0 since 17.1 relies on react-native 38 but we are still on 37. To remove also the `whatwg-fetch` warning you could include it as well as devDependency: 

```
"jest-react-native": "~17.0.2",
"whatwg-fetch": "~2.0.0"
```

**Comment by tino-junge** [comment]
@Lapanti You can now rebase on master where we set now "jest-react-native": "~17.0.2"

**Pull request merged**